modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceFacets.testcase

```diff
@@ -14,6 +14,96 @@ definition {
 		CommerceConfiguration.commerceTearDown();
 	}
 
+	@description = "This is a test case for COMMERCE-10907. User can set custom Price Range Facet configurations and use Price Range Facet widget to filter the products, the results should be visible in the Search Result widget"
+	@priority = 3
+	test AssertProductsAreFilteredAfterUpdatingRangesByPriceRangeFacet {
+		property portal.acceptance = "false";
+
+		task ("Given the first two ranges are changed on Price Range Facet widget on a Minium site") {
+			Navigator.openWithAppendToBaseURL(urlAppend = "web/minium/catalog");
+
+			var rangesArray = '''[{'range': '[0 TO 59.99]'}\, {'range': '[60 TO 99.99]'}\, {'range': '[100 TO 199.99]'}\, {'range': '[200 TO 499.99]'}\, {'range': '[500 TO 999.99]'}\, {'range': '[1000 TO *]'}]''';
+
+			CommerceWidget.updateFacetConfiguration(
+				portletName = "Price Range Facet",
+				rangesArray = ${rangesArray});
+		}
+
+		task ("When refresh the page") {
+			Refresh();
+		}
```

Comment:
Is it necessary to refresh to see the updated ranges? If it is, let's keep the refresh

Reply:
@lorenzolapescara, for this bug, it was reproduced with refresh action. So if we don't use Refresh function, I think we should use another way like navigation by URL to keep covering this scenario. Which way do you think is better?

Reply:
@lorenzolapescara, for this bug, it was reproduced with refresh action. So if we don't use Refresh function, I think we should use another way like navigation by URL to keep covering this scenario. Which way do you think is better?